packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -143,16 +137,38 @@ export default function WorkflowListPage({ workflows, profileOptions, hasNoWorkf
                             )}
                           </Badge>
                         </li>
-                        {workflow.teamId && (
-                          <li>
-                            <Badge variant="gray">
-                              <>{workflow.team?.name}</>
-                            </Badge>
-                          </li>
-                        )}
+                        <div className="block md:hidden">
+                          {workflow.team?.name && (
+                            <li>
+                              <Badge variant="gray">
+                                <>{workflow.team.name}</>
+                              </Badge>
+                            </li>
+                          )}
+                        </div>
                       </ul>
                     </div>
                   </Link>
+                  <div>
+                    <div className="hidden md:block">
+                      {workflow.team?.name && (
+                        <Badge className="mr-4 mt-1 p-[1px] px-2" variant="gray">
+                          <Avatar
+                            alt={workflow.team?.name || ""}
+                            href={
+                              workflow.team?.id
+                                ? `/settings/teams/${workflow.team?.id}/profile`
+                                : "/settings/my-account/profile"
+                            }
+                            imageSrc={`${WEBAPP_URL}/${workflow.team?.slug}/avatar.png` || undefined}
```

Comment:
We should use the URL being used to access this page which could include a subdomain giving the avatar the context of an organization to narrow down the avatar to return.

Reply:
What would be the right way to access this URL? And wouldn't we need to change that in a lot of other places as well?

Reply:
You can get the org slug from the Org Branding provider and construct it like this  `${orgBranding.slug}.${subdomainSuffix()}` using 
`subdomainSuffix` from "@calcom/features/ee/organizations/lib/orgDomains". It will be probably a good idea to include this to utils somewhere too, you can search that it is being used in more places.